robotExecution/test.robot

```diff
@@ -1,7 +1,24 @@
 *** Settings ***
```

Comment:
Not a fan of having to have to run actual RPAf code for testing. Not sure how this would behave on different machines, as i.e. I am having problems with RPAf execution on my machine. The test should still pass because when the functionality works and should not depend on outside factors. Maybe try to mock or emulate values.

Reply:
This is actually a debugging robot, so that one does not always have to configure a robot in the web application to test the functionality.  There is also a comment for that in `socketConnection.js`: 
`// test.robot is just used for testing purposes. Use executable.robot for the real product.`
There is no test to check the robot framework execution apart from try catch blocks. In my opinion that's also not necessary as robot framework by itself is test by itself.

Reply:
In that case, please rename the robot into (for example) debugging.robot and document its functionality more in detail, so that any person picking up rom the point where we left of can understand it and knows how to use it for debugging purposes